src/app/(main)/legal/[id]/page.tsx

```diff
@@ -20,7 +20,7 @@ async function Page({ params }: Props) {
   const { perspective, isDraftMode } = getDraftModeInfo();
 
   const initialDocument = await loadStudioQuery<LegalDocument>(
-    LEGAL_DOCUMENTS_BY_SLUG_AND_LANG_QUERY,
+    LEGAL_DOCUMENT_BY_SLUG_AND_LANG_QUERY,
     { slug: id, language: "en" }, //TODO: replace this with selected language for the page
```

Comment:
We could probably update this hard coded variable as well.

Reply:
This file will be removed by #734 , so didn't bother to update it here 🙃

Reply:
You're right!